librz/core/cmd/cmd_debug.c

```diff
@@ -4410,23 +4406,65 @@ RZ_IPI RzCmdStatus rz_cmd_debug_add_bp_handler(RzCore *core, int argc, const cha
 
 // dbl
 RZ_IPI RzCmdStatus rz_cmd_debug_list_bp_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
-	RzOutputMode mode = state->mode;
-	switch (mode) {
-	case RZ_OUTPUT_MODE_STANDARD:
-		rz_bp_list(core->dbg->bp, 0);
-		break;
-	case RZ_OUTPUT_MODE_RIZIN:
-		rz_bp_list(core->dbg->bp, 1);
-		break;
-	case RZ_OUTPUT_MODE_JSON:
-		rz_bp_list(core->dbg->bp, 'j');
-		break;
-	case RZ_OUTPUT_MODE_QUIET:
-		rz_bp_list(core->dbg->bp, -1);
-		break;
-	default:
-		rz_warn_if_reached();
-		break;
+	RzBreakpointItem *b;
+	RzListIter *iter;
+	RzOutputMode mode = RZ_OUTPUT_MODE_STANDARD;
+	PJ *pj = NULL;
+	if (state) {
```

Comment:
remove this check and just use a `rz_return_val_if_fail(state, RZ_CMD_STATUS_ERROR)` at the beginning of the function.